<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>omni::ui::MainWindow</class>
 <widget class="QMainWindow" name="omni::ui::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>539</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Omnidome</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabContainer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="pages" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>11</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="grpProjectors" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="omni::ui::proj::TuningList" name="tuningList" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnAddTuning">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Add another projector</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockBlend">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>BLEND</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="omni::ui::Blend" name="dockBlendWidget"/>
  </widget>
  <widget class="QDockWidget" name="dockWarp">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>WARP</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="omni::ui::Warp" name="dockWarpWidget"/>
  </widget>
  <widget class="QDockWidget" name="dockCanvas">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>22</width>
     <height>200</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>CANVAS</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="omni::ui::Canvas" name="dockCanvasWidget"/>
  </widget>
  <widget class="QDockWidget" name="dockInputs">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>INPUT</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="omni::ui::Input" name="dockInputsWidget"/>
  </widget>
  <widget class="QDockWidget" name="dockMapping">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>MAPPING</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="omni::ui::Mapping" name="dockMappingWidget"/>
  </widget>
  <widget class="QDockWidget" name="dockScene">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string notr="true">SCENE</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="omni::ui::Scene" name="dockSceneWidget"/>
  </widget>
  <widget class="QDockWidget" name="dockColorCorrection">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>COLOR ADJUSTMENT</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="omni::ui::ColorCorrection" name="dockColorCorrectionWidget"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionEditAsNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionRecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDisableAllTunings"/>
    <addaction name="actionEnableAllTunings"/>
    <addaction name="actionEnableSelectedTuningOnly"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New calibration</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open a calibration from file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save calibration</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionEditAsNew">
   <property name="text">
    <string>Edit As New</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionLast">
   <property name="text">
    <string>Last</string>
   </property>
  </action>
  <action name="actionDisableAllTunings">
   <property name="text">
    <string>Turn all projectors off</string>
   </property>
  </action>
  <action name="actionEnableAllTunings">
   <property name="text">
    <string>Turn all projectors on</string>
   </property>
  </action>
  <action name="actionEnableSelectedTuningOnly">
   <property name="text">
    <string>Show selected projector only</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRecentFiles">
   <property name="text">
    <string>Recent files</string>
   </property>
  </action>
  <action name="action10_fps">
   <property name="text">
    <string>10 fps</string>
   </property>
  </action>
  <action name="action25_fps">
   <property name="text">
    <string>15 fps</string>
   </property>
  </action>
  <action name="action5_fps">
   <property name="text">
    <string>5 fps</string>
   </property>
  </action>
  <action name="action60_fps">
   <property name="text">
    <string>60 fps</string>
   </property>
  </action>
  <action name="action30_fps">
   <property name="text">
    <string>30 fps</string>
   </property>
  </action>
  <action name="actionNo_limit">
   <property name="text">
    <string>no limit</string>
   </property>
  </action>
  <action name="actionProjector_View_FPS">
   <property name="text">
    <string>Projector View FPS</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>omni::ui::proj::TuningList</class>
   <extends>QWidget</extends>
   <header location="global">proj/TuningList.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>omni::ui::Canvas</class>
   <extends>QWidget</extends>
   <header>Canvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>omni::ui::Blend</class>
   <extends>QWidget</extends>
   <header>Blend.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>omni::ui::Warp</class>
   <extends>QWidget</extends>
   <header>Warp.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>omni::ui::Input</class>
   <extends>QWidget</extends>
   <header>Input.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>omni::ui::Mapping</class>
   <extends>QWidget</extends>
   <header>Mapping.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>omni::ui::Scene</class>
   <extends>QWidget</extends>
   <header>Scene.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>omni::ui::ColorCorrection</class>
   <extends>QWidget</extends>
   <header>ColorCorrection.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rc/omnidome.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>omni::ui::MainWindow</receiver>
   <slot>newProjection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>omni::ui::MainWindow</receiver>
   <slot>openProjection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>omni::ui::MainWindow</receiver>
   <slot>saveProjection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditAsNew</sender>
   <signal>triggered()</signal>
   <receiver>omni::ui::MainWindow</receiver>
   <slot>editAsNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>omni::ui::MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveAs</sender>
   <signal>triggered()</signal>
   <receiver>omni::ui::MainWindow</receiver>
   <slot>saveProjectionAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newProjection()</slot>
  <slot>openProjection()</slot>
  <slot>saveProjection()</slot>
  <slot>editAsNew()</slot>
  <slot>addTuning()</slot>
  <slot>setMode(int)</slot>
  <slot>showSettings()</slot>
  <slot>saveProjectionAs()</slot>
 </slots>
</ui>
